cmd/snap-confine/mount-support.c

```diff
@@ -638,6 +652,8 @@ void sc_populate_mount_ns(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 			{"/media", true},	// access to the users removable devices
 #endif				// MERGED_USR
 			{"/run/netns", true},	// access to the 'ip netns' network namespaces
+			{"/mnt",.is_optional = true},	// to support the removable-media interface
```

Comment:
Nitpicks land, but spacing here looks a little odd. Might be a bit easier to read as:
```
{"/mnt", .is_optional=true}

Reply:
Yeah, its odd - this comes from `make fmt` which uses indent which apparently has this strange formatting.